core/src/bitmap/turbulence.rs

```diff
@@ -38,9 +39,11 @@ fn random(seed: i64) -> i64 {
 
 #[derive(Copy, Clone)]
 struct StitchInfo {
-    width: i32, // How much to subtract to wrap for stitching.
+    /// How much to subtract to wrap for stitching.
+    width: i32,
     height: i32,
-    wrap_x: i32, // Minimum value to wrap.
+    /// Minimum value to wrap.
+    wrap_x: i32,
     wrap_y: i32,
```

Comment:
Isn't this going to be weird that the doc comments are only for the first half of each pair? :thinking:

Reply:
Oh, I didn't notice those comment referred to more than one field, TBH it was weird before too... :/

I can move those comments to the doc comment of the struct or add x/y width/height variants (or something else perhaps?), which one do you prefer?

Reply:
Done

Reply:
Thanks! ^^